kubetest/main.go

```diff
@@ -418,7 +420,11 @@ func complete(o *options) error {
 func acquireKubernetes(o *options) error {
 	// Potentially build kubernetes
 	if o.build.Enabled() {
-		if err := xmlWrap("Build", o.build.Build); err != nil {
+		err := xmlWrap("Build", o.build.Build)
+		if o.flushMemAfterBuild {
```

Comment:
maybe call it inside build.go:`(b *buildStrategy) Build()` and `(b *buildFederationStrategy) Build()`

Reply:
they don't have access to the options

Reply:
you can set it as a var, like `verbose`, rather an option? Or `build.go` can take in flags but which is a `/shrug`

Reply:
- I don't think *more* global var is better.
- xmlWrap doesn't allow passing variables so there would have to be an anonymous wrapper around Build() here here that passed the variable in which is not an improvement.